lib/module.php

```diff
@@ -181,6 +182,12 @@ function solusiovps_CreateAccount(array $params): string
     }
 
     try {
+        $params['password'] = Strings::generatePassword();
+        $results = localAPI('EncryptPassword', ['password2' => $params['password']]);
```

Comment:
May be it make sense to wrap this method with ours, like `Crypt::encrypt(string $str): string`?